redbot/cogs/trivia/trivia.py

```diff
@@ -402,6 +405,64 @@ async def trivia_list(self, ctx: commands.Context):
             else:
                 await ctx.send(msg)
 
+    @trivia.command(name="info")
+    async def trivia_info(self, ctx: commands.Context, category: str.lower):
+        """Get information about a trivia category."""
+        try:
+            data = self.get_trivia_list(category)
+        except FileNotFoundError:
+            return await ctx.send(
+                _(
+                    "Invalid category `{name}`. See `{prefix}trivia list` for a list of "
+                    "trivia categories."
+                ).format(name=category, prefix=ctx.clean_prefix)
+            )
+        except InvalidListError:
+            return await ctx.send(
+                _(
+                    "There was an error parsing the trivia list for the `{name}` category. It "
+                    "may be formatted incorrectly."
+                ).format(name=category)
```

Comment:
```suggestion
                _(
                    "There was an error parsing the trivia list for the {name} category."
                    " It may be formatted incorrectly."
                ).format(name=inline(category))
```